src/components/StepperDialog/StepperDialog.jsx

```diff
@@ -25,13 +25,8 @@ const useStyles = makeStyles(() => ({
   }
 }))
 
-// TODO Test and improve it & PR cozy-ui
-const StepperDialog = props => {
+const StepperDialog = ({ onClose, title, content, stepper, ...rest }) => {
```

Comment:
what kind of version, do you want to create?
is it a patch, a minor?

are you sure of the commit name? 🤔